scripts/lib/BS.pm

```diff
@@ -253,6 +253,14 @@ sub menu {
 
     # push @{$menu}, $self->_main_menu_trading();
 
+    push @menu,
+        {
+        id         => 'topMenuIcoBids',
+        class      => 'ja-hide client_logged_in',
+        url        => url_for('/user/ico-subscribe'),
+        text       => localize('ICO Bids'),
```

Comment:
@4p00rv need to add the translation string in `GenerateStaticData.pm` file. also can we move the comment above this menu item instead of below it? :)